content/v1/csidriver/features/powerscale.md

```diff
@@ -341,30 +344,35 @@ spec:
 
 This manifest creates a pod in a given cluster and attaches a newly created ephemeral inline CSI volume to it.
 
+**Note**: Storage class is not supported in CSI ephemeral inline volumes and all parameters are driver specific.
+CSI ephemeral volumes allow users to provide volumeAttributes directly to the CSI driver as part of the Pod spec.
+Following `volumeAttributes` are supported: size, ClusterName, AccessZone, IsiPath, IsiVolumePathPermissions, AzServiceIP.
+For reference, check the description of parameters in the following example: [isilon.yaml](https://github.com/dell/csi-powerscale/blob/main/samples/storageclass/isilon.yaml)
+
 ## Topology
+
 ### Topology Support
 
-The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist. 
+The CSI PowerScale driver version 1.4.0 and later supports Topology by default which forces volumes to be placed on worker nodes that have connectivity to the backend storage, as a result of which the nodes which have access to PowerScale Array are appropriately labeled. The driver leverages these labels to ensure that the driver components (controller, node) are spawned only on nodes wherein these labels exist.
```

Comment:
Remove comma after "storage" and add a period (hard stop). Change "as a result of which" to "This results in". Change "are" to "being"